dash/include/dash/GlobRef.h

```diff
@@ -61,12 +61,14 @@ class GlobRef
 
 public:
   /**
+   * TODO: Reference semantics forbid declaration without definition.
+   *
    * Default constructor, creates an GlobRef object referencing an element in
    * global memory.
    */
   GlobRef()
-  : _gptr(DART_GPTR_NULL) {
-  }
+  : _gptr(DART_GPTR_NULL)
+  { }
```

Comment:
That's what the TODO comment refers to. There are situations where GloRef objects are temporarily uninitialized. Perhaps I can eliminate them.